cli/src/main/java/com/devonfw/tools/ide/process/ProcessContextImpl.java

```diff
@@ -171,6 +176,30 @@ public ProcessResult run(ProcessMode processMode) {
     }
   }
 
+  /**
+   * Asynchronously and parallel reads {@link InputStream input stream} and stores it in {@link CompletableFuture}.
+   * Taken from:
+   * https://stackoverflow.com/questions/14165517/processbuilder-forwarding-stdout-and-stderr-of-started-processes-without-blocki/57483714#57483714
+   * 
+   * @param is {@link InputStream}.
+   * @return {@link CompletableFuture}.
+   */
+  private static CompletableFuture<String> readInputStream(InputStream is) {
+
+    return CompletableFuture.supplyAsync(() -> {
+      try (InputStreamReader isr = new InputStreamReader(is); BufferedReader br = new BufferedReader(isr);) {
+        StringBuilder res = new StringBuilder();
+        String inputLine;
+        while ((inputLine = br.readLine()) != null) {
+          res.append(inputLine).append(System.lineSeparator());
```

Comment:
You changed the existing behavior by adding a lineSeparator.
This broke the tests:
```
[ERROR] Tests run: 3, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.086 s <<< FAILURE! -- in com.devonfw.tools.ide.tool.jasypt.JasyptTest
[ERROR] com.devonfw.tools.ide.tool.jasypt.JasyptTest.testJasyptRun -- Time elapsed: 0.045 s <<< FAILURE!
java.lang.AssertionError: 
[INFO-Log messages] 
Expecting LinkedList:
  ["executing java:
This is a jar file.
"]
to contain:
  ["executing java:"]
but could not find the following element(s):
  ["executing java:"]
```

Reply:
Thanks, should be fixed now.